README.md

```diff
@@ -26,6 +26,15 @@ Read more on the [Confluence space](https://dsdmoj.atlassian.net/wiki/spaces/MRD
 * [Deployment / Helm](./docs/helm-deploy.md) - including how to roll back a deployment
 * [Runbook](./RUNBOOK.md)
 
+### Maintenance Banner
+* To display a maintenance banner to make users aware of any planned outages five environment variables are required. These are available for each environment in the helm_deploy folder -
+  * **SHOW_NOTIFICATION** - this must be set to true for the banner to manifest, by default it is false
```

Comment:
How does this interact with the start date and end date?

Reply:
This is a sort of master switch. If SHOW_NOTIFICATION is false (default), the banner won't be displayed regardless of start/end dates. Andrew's old 'Assess risks and needs' project follows the same pattern.